pilot/pkg/security/authz/policy/helper.go

```diff
@@ -177,7 +178,7 @@ func SimpleAuthzPolicy(name string, namespace string) *model.Config {
 					From: []*authpb.Rule_From{
 						{
 							Source: &authpb.Source{
-								Principals: []string{AuthzPolicyTag(name)},
+								Principals: principals,
```

Comment:
Can we not change this? If you need fake authz policy for some principals, you can create a new function for that. The reason here is the change is breaking the convention we were trying to maintain in these unit tests. i.e. `SimpleAuthzPolicy` returns a placeholder of the policy that is unique and distinguishable by its name/namespace.